src/bun.js/bindings/exports.zig

```diff
@@ -2585,14 +2754,29 @@ pub const ZigConsoleClient = struct {
                             this.printAs(.Class, Writer, writer_, value, jsType, enable_ansi_colors)
                         else if (value.isCallable(this.globalThis.vm()))
                             this.printAs(.Function, Writer, writer_, value, jsType, enable_ansi_colors)
-                        else
-                            writer.writeAll("{}");
-                    } else {
-                        if (iter.always_newline) {
-                            this.indent -|= 1;
+                        else {
```

Comment:
This code should only run with snapshot format right?

Reply:
updated